drivers/gpio/gpio_b9x.c

```diff
@@ -651,10 +659,26 @@ static int gpio_b9x_pm_action(const struct device *dev, enum pm_device_action ac
 				reg_irq_risc1_en(GET_PORT_NUM(gpio))
 				= data->gpio_b9x_retention.risc1_irq_conf;
 
+				irq_num -= CONFIG_2ND_LVL_ISR_TBL_OFFSET;
```

Comment:
Please add comment with explanation about setting pending IRQ.
Without it seems starange

Reply:
Done